lib/srv/regular/sshserver.go

```diff
@@ -1264,20 +1264,24 @@ func (s *Server) HandleNewConn(ctx context.Context, ccx *sshutils.ConnectionCont
 	}
 
 	// Create host user.
-	created, userCloser, err := s.termHandlers.SessionRegistry.TryCreateHostUser(identityContext)
+	created, userCloser, err := s.termHandlers.SessionRegistry.UpsertHostUser(identityContext)
```

Comment:
Dropping the `Try` nomenclature since these functions return errors. Eventually I'd like to change the public interface of `usermgmt` to avoid this confusion, but for now renaming it is easy